notebooks/k_means.livemd

```diff
@@ -5,13 +5,14 @@
 ```elixir
 Mix.install([
   {:scholar, github: "elixir-nx/scholar", override: true},
-  {:exla, "~> 0.5.3"},
-  {:nx, github: "elixir-nx/nx", sparse: "nx", override: true},
+  {:exla, "~> 0.6.0"},
+  {:nx, "~> 0.6.0", override: true},
```

Comment:
what needs this override?

Reply:
```
Because your app depends on explorer ~> 0.6.1 which depends on nx ~> 0.4.0 or ~> 0.5.0, nx ~> 0.4.0 or ~> 0.5.0 is required.
So, because your app depends on nx ~> 0.6.0, version solving failed.
```

Reply:
Makes sense. Though maybe if explorer is ready, allowing for more recent Nx could be nice

Reply:
We will
Ship new
Version soon

Reply:
I'll change this as soon as the new version of Explorer will be released